package.json

```diff
@@ -50,19 +49,11 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run precommit"
+      "pre-commit": "lint-staged"
```

Comment:
Will this run flow? The old npm run precommit does.

Reply:
Oh, yeah, we need to put this line back the way it was. Good catch.